spec/concerns/authenticator_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+shared_examples_for "authenticator" do
+  let(:model) { FactoryBot.create(described_class.to_s.underscore.to_sym) }
+
+  describe "Instance methods" do
+    describe "response to instance methods" do
+      [:verify_email, :activated?, :authenticated_token?, :set_reset_digest, :send_password_reset_email, :password_reset_expired?].each do |instance_method|
+        it "responds to #{ instance_method }" do
+          expect(model).to respond_to(instance_method)
+        end
+      end
+    end
+    describe "verify_email" do
+      it "updates verification_token column of model" do
+        model.verify_email
```

Comment:
Make use of `before` block for such statements

Reply:
This file should be stored as `spec/models/concerns/authenticator_spec.rb` because it is a model concern.